<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>995</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="qCustomPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>801</width>
      <height>721</height>
     </rect>
    </property>
    <widget class="QSpinBox" name="data_spinBox">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>0</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>447</x>
       <y>5</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>data</string>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="qCustomPlot2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>591</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="QSpinBox" name="angle_spinBox">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>angle</string>
     </property>
    </widget>
   </widget>
   <widget class="QDoubleSpinBox" name="gainnumber">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="gainlabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>增益（dB）</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="vol">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="vollabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前声速（m/s）</string>
    </property>
   </widget>
   <widget class="QComboBox" name="materialbox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentText">
     <string>铁</string>
    </property>
    <item>
     <property name="text">
      <string>铁</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>铝</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>木</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="materials">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
